src/services/static-snapshot-service.ts

```diff
@@ -0,0 +1,20 @@
+import logger from '../utils/logger'
+import {StaticSnapshotOptions} from './static-snapshot-options'
+
+export default class StaticSnapshotService {
```

Comment:
As I read through this, I wonder if it would make sense to have the snapshot service extend the agent service? Or at least take care of the setup of the agent service in this constructor, so you don't have to do that in the `run` command method

Reply:
Hmm, I feel like this conversation/decision might be better had on #137 since that fleshes out (and changes) the static snapshot service @Robdel12